deploy/examples/create-external-cluster-resources.py

```diff
@@ -205,6 +207,41 @@ def gen_arg_parser(cls, args_to_parse=None):
             args_to_parse = sys.argv[1:]
         return argP.parse_args(args_to_parse)
 
+    def validate_rgw_metadata_ec_pool_name(self):
+        if self._arg_parser.rbd_metadata_ec_pool_name:
+            rbd_metadata_ec_pool_name = self._arg_parser.rbd_metadata_ec_pool_name
+            rbd_pool_name = self._arg_parser.rbd_data_pool_name
```

Comment:
We must check that `rbd_pool_name` is an EC pool, you can use `erasure_code_profile` from the output.

Reply:
I think this condition will verify this right https://github.com/rook/rook/blob/2aeba6231da2669db373d5fde8c95cb286960675/deploy/examples/create-external-cluster-resources.py#L234-L235